src/calculateRentalCost.test.js

```diff
@@ -1,23 +1,19 @@
 'use strict';
 
-const calculateRentalCost = require('./calculateRentalCost');
+const calculateRentalCost = require('./calculateRentalCost ');
```

Comment:
There is an extra space at the end of the require path './calculateRentalCost '. This might cause issues when running the tests as the module path should be exact without any trailing spaces.